Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -4517,7 +4517,7 @@ void LocalFrameView::updateScrollCorner()
     std::unique_ptr<RenderStyle> cornerStyle;
     IntRect cornerRect = scrollCornerRect();
     
-    if (!cornerRect.isEmpty()) {
+    if (!cornerRect.isEmpty() && scrollbarWidthStyle() == ScrollbarWidth::Auto && !scrollbarColorStyle().has_value()) {
```

Comment:
Check that this pseudo isn't used anywhere else (RenderLayerScrollableArea potentially?)

Potentially move this to its own patch and add a test that explicitly covers this scenario.

At the very least this can potentially be extracted into RenderStyle like we have for "usesLegacyScrollbarStyle" (investigate if that function would already guard against this)

Reply:
Moved into its own patch https://github.com/WebKit/WebKit/pull/15264